### ISSUE TITLE
Problems

### ISSUE THREAD

**Issue by lorrp1** [AminHP/gym-anytrading#29]
A

**Comment by lorrp1** [comment]
this is using the example:
![Screenshot from 2020-10-16 18-40-44](https://user-images.githubusercontent.com/15959054/96285437-3f5b3e00-0fdf-11eb-83bc-5c9bfd41e561.png)

**Comment by AminHP** [comment]
It seems there is a problem with `stable_baselines`. I tried to implement a DQN from scratch, and it works just fine with the sin function.

**Comment by toksis** [comment]
Can you change the learning rate? 

```python
model = DQN(MlpPolicy, env, verbose=1,learning_rate=0.00001)?
```
I tried to do a learning rate of 1. and it do this.

![image](https://user-images.githubusercontent.com/5316255/96378273-494f8d80-11e7-11eb-8429-5997df24615a.png)

**Comment by toksis** [comment]
Learning rate of .0001

![image](https://user-images.githubusercontent.com/5316255/96381552-8ff3b680-11ec-11eb-8bb4-b5ddf562adf1.png)

**Comment by lorrp1** [comment]
@AminHP i used stable-baseline3 which uses torch since i cant use the old version, the requirement is ts <1.15, i have the 2.x instead.
im going to try with RLlib

**Comment by toksis** [comment]
Im using the old stable baseline.

**Comment by toksis** [comment]
Maybe you can share your sin function code? @lorrp1

**Comment by lorrp1** [comment]
@AminHP  im getting even worse results with ray

**Comment by AminHP** [comment]
I suggest you read [this](https://medium.com/hackernoon/the-self-learning-quant-d3329fcc9915#.3b4ghaoa7) article and implement your own DQN and test it with the environment.

**Comment by lorrp1** [comment]
@toksis 
df = pd.DataFrame({'timestamp':range(1,10001), 'close': 1.1+np.sin(np.arange(1,10001)/50) * 10 + np.arange(1,10001)/50})
and example of sort of fx= sin(x) + x
@AminHP 
but are you able to use a2c of stable baseline with the sin? 
im using the torch version (v3 beta) so maybe that's why im getting bad results

**Comment by toksis** [comment]
try the zoo in stable baselines.

**Comment by lorrp1** [comment]
@toksis  are you able to get it working with the sin function there?

**Comment by toksis** [comment]
Sadly no... I want to try the zoo in stable baselines maybe it will help. But first I will try the article from AdminHP.
